src/integration/response-quality.test.ts

```
@@ -29,107 +29,190 @@ describeIntegration('Response Quality Validation', () => {
       expect(response).toContain('Swift by Sundell');
       expect(response).toContain('Antoine van der Lee');
+      expect(response).toContain('Nil Coalescing');
+      expect(response).toContain('Point-Free');
     });
 
-    it('should include actionable instructions', async () => {
+    it('should show status indicators for each source', async () => {
+      const response = await client.callToolText('list_content_sources');
+
+      // Each source should have a status indicator
+      expect(response).toMatch(/[✅⚙️⬜]/);
```

Biome (2.1.2)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)